### PR TITLE
Pre-process LESS example

### PR DESCRIPTION
**Pull request by gregorypratt**

### CODE REVIEW

**Comment by gregorypratt** [comment]
Resubmitted to gh-pages as per request!

File: pre-process-less/index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  <script src="message-two.tag" type="riot/tag"></script>
+
+  <!-- scripts we need -->
+  <script src="https://cdn.jsdelivr.net/less/2.2.0/less.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/g/riot@2.2(riot.min.js+compiler.min.js)"></script>
+
+  <!-- mount this app -->
+  <script>
+    riot.parsers.css.less = function(tagName, css) {
+      var style = '';
```
Comment (cognitom) [inline pre-process-less/index.html]: No semicolon pls.

**Comment by cognitom** [comment]
@gregorypratt Thank you for your patience!

File: pre-process-less/index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  <script src="message-one.tag" type="riot/tag"></script>
+  <script src="message-two.tag" type="riot/tag"></script>
+
+  <!-- scripts we need -->
+  <script src="https://cdn.jsdelivr.net/less/2.2.0/less.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/g/riot@2.2(riot.min.js+compiler.min.js)"></script>
+
+  <!-- mount this app -->
+  <script>
+    riot.parsers.css.less = function(tagName, css) {
```
Comment (cognitom) [inline pre-process-less/index.html]: Sorry, a small point. The parameter `css` seems containing LESS...? Would it be more readable to change?
- `css` --> `style`
- `style` --> `css`

Comment (gregorypratt) [inline pre-process-less/index.html]: Do you mean like this:

``` javascript
riot.parsers.css.less = function(tagName, less) {
  var style = ''
  less.render(less, {}, function(error, output) {
    style = output.css
  });
  return style
}
```

File: pre-process-less/index.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  <script src="message-one.tag" type="riot/tag"></script>
+  <script src="message-two.tag" type="riot/tag"></script>
+
+  <!-- scripts we need -->
+  <script src="https://cdn.jsdelivr.net/less/2.2.0/less.min.js"></script>
+  <script src="https://cdn.jsdelivr.net/g/riot@2.2(riot.min.js+compiler.min.js)"></script>
+
+  <!-- mount this app -->
+  <script>
+    riot.parsers.css.less = function(tagName, style) {
```
Comment (gregorypratt) [inline pre-process-less/index.html:24]: Better? "Less" code...

File: pre-process-less/index.html (hunk 2 above)
Comment (cognitom) [inline pre-process-less/index.html]: How about this? `less` is already assign to the compiling method. 

``` javascript
riot.parsers.css.less = function(tagName, style) {
  var css = ''
  less.render(style, {}, function(error, output) {
    css = output.css
  });
  return css
}
```

File: pre-process-less/index.html (hunk 3 above)
Comment (cognitom) [inline pre-process-less/index.html:24]: I don't think so...
It looks not straightforward. Thoughts?

Comment (gregorypratt) [inline pre-process-less/index.html:24]: I think we're over thinking it! Nevertheless I've updated it ;-)

Comment (cognitom) [inline pre-process-less/index.html:24]: Yeah, agreed. Thank you for your update :+1:

**Pull request merged**